modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/Tabs.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayTabs from '@clayui/tabs';
+import React from 'react';
+
+export default function Tabs({
+	activation,
+	additionalProps: _additionalProps,
+	componentId: _componentId,
+	cssClass,
+	displayType,
+	fade,
+	justified,
+	locale: _locale,
+	panels,
+	portletId: _portletId,
+	portletNamespace: _portletNamespace,
+	tabsItems,
+	...otherProps
+}) {
+
+	// TO REMOVE ONCE CLAY IS RELEASED
+
+	const activeIndex = tabsItems.indexOf(
+		tabsItems.find((item) => item.active)
+	);
+
+	return (
+		<>
+			<ClayTabs
+				activation={activation}
+				className={cssClass}
+				defaultActive={activeIndex}
+				displayType={displayType}
+				fade={fade}
+				justified={justified}
+				{...otherProps}
```

Comment:
How come `displayType`, `fade`, and `justified` are all passed directly through, but then also opaquely passing through `otherProps`?

I'm not totally against it, just curious why some props were added one way and some another

Reply:
I usually make explicit the props defined in the tag (like fade, etc) and leave those that are not to be passed through otherProps (like any other additionalProps defined by the user)

Reply:
Ah I gotcha, sounds good!